data/en/test_question_definition.json

```diff
@@ -30,10 +30,11 @@
                     "id": "question",
                     "title": "Do you connect a LiFePO4 battery to your <em>photovoltaic system</em> to store surplus energy?",
                     "type": "General",
-                    "description": "",
                     "definitions": [{
                         "title": "What is a photovoltaic system?",
-                        "description": "A typical photovoltaic system employs solar panels, each comprising a number of solar cells, which generate electrical power. PV installations may be ground-mounted, rooftop mounted or wall mounted. The mount may be fixed, or use a solar tracker to follow the sun across the sky."
+                        "content": [{
+                            "description": "A typical photovoltaic system employs solar panels, each comprising a number of solar cells, which generate electrical power. PV installations may be ground-mounted, rooftop mounted or wall mounted. The mount may be fixed, or use a solar tracker to follow the sun across the sky."
```

Comment:
What about `titles` and `list`?

Reply:
Fair point, will update tests. Currently, have tests for definition but ya only for description. 👍